src/main/resources/application.yml

```diff
@@ -0,0 +1,22 @@
+spring:
+  application:
+    name: service-template
+  datasource:
+    url: ${DB_URL}
+    username: ${DB_USERNAME}
+    password: ${DB_PASSWORD}
+    driver-class-name: org.postgresql.Driver
+    jpa:
+      database: postgresql
+      database-platform: org.hibernate.dialect.PostgreSQL10Dialect
+grpc:
+  server:
+    port: 9090
+  client:
+    grpc-server:
+      address: static://localhost:9090
+      negotiationType: PLAINTEXT
+      enableKeepAlive: true
+      keepAliveWithoutCalls: true
+      keepAliveTime: 60s
+      keepAliveTimeout: 20s
```

Comment:
More like a matter of taste but I found it a bit difficult to use yaml sometimes when you need to search for property usages over the codebase. WDYT about using plain-old .properties?

Reply:
fixed